resources/settings.xml

```diff
@@ -1,132 +1,132 @@
+    <setting id="viewmodemovie" type="select" label="30040" lvalues="30162|30142" default="0" enable="eq(-9,true)"/>
+    <setting id="viewmodemovieid" type="number" label="30143" visible="eq(-1,1) + eq(-10,true)" enable="eq(-10,true)" subsetting="true"/>
+    <setting id="viewmodeshow" type="select" label="30041" lvalues="30162|30142" default="0" enable="eq(-11,true)"/>
+    <setting id="viewmodeshowid" type="number" label="30143" visible="eq(-1,1) + eq(-12,true)" enable="eq(-12,true)" subsetting="true"/>
+    <setting id="viewmodeseason" type="select" label="30042" lvalues="30162|30142" default="0" enable="eq(-13,true)"/>
+    <setting id="viewmodeseasonid" type="number" label="30143" visible="eq(-1,1) + eq(-14,true)" enable="eq(-14,true)" subsetting="true"/>
+    <setting id="viewmodeepisode" type="select" label="30043" lvalues="30162|30142" default="0" enable="eq(-15,true)"/>
+    <setting id="viewmodeepisodeid" type="number" label="30143" visible="eq(-1,1) + eq(-16,true)" enable="eq(-16,true)" subsetting="true"/>
+    <setting id="viewmodesearch" type="select" label="30160" lvalues="30162|30142" default="0" enable="eq(-17,true)"/>
+    <setting id="viewmodesearchid" type="number" label="30143" visible="eq(-1,1) + eq(-18,true)" enable="eq(-18,true)" subsetting="true"/>
+    <setting id="viewmodeexported" type="select" label="30161" lvalues="30162|30142" default="0" enable="eq(-19,true)"/>
+    <setting id="viewmodeexportedid" type="number" label="30143" visible="eq(-1,1) + eq(-20,true)" enable="eq(-20,true)" subsetting="true"/>
+  </category>
+  <category label="30078"><!--Playback-->
+    <setting id="BookmarkManager_enabled" type="bool" label="30083" visible="false" default="true"/>
+    <setting id="StreamContinuityManager_enabled" type="bool" label="30082" default="true"/>
+    <setting id="SectionSkipper_enabled" type="bool" label="30075" default="true"/>
+    <setting id="auto_skip_credits" type="bool" label="30079" default="false" visible="eq(-1,true)" subsetting="true"/>
+    <setting id="pause_on_skip" type="bool" label="30080" default="false" visible="eq(-1,true)" subsetting="true"/>
+    <setting id="UpNextNotifier_enabled" type="bool" label="30129" default="true"/>
+    <setting id="is_settings_upnext" type="action" label="30178" action="Addon.OpenSettings(service.upnext)" enable="System.HasAddon(service.upnext)" option="close" visible="eq(-1,true)" subsetting="true"/>
+    <setting id="ResumeManager_enabled" type="bool" label="30176" default="true"/>
+    <setting id="ResumeManager_dialog" type="bool" label="30177" default="true" visible="eq(-1,true)" subsetting="true"/>
+    <setting id="forced_subtitle_workaround" type="bool" label="30181" default="true" />
+  </category>
+  <category label="30023"><!--Expert-->
+    <setting id="StreamType" type="enum" label="30195" lvalues="30196|30197|30198|30199|30200|30201" default="0"/>

```

Comment:
Default is 0, so SD. People may complain about quality.
It should be 4, unless the setting is not mandatory.

Reply:
I think we should set it to max...so 4k, or I can add a Max Option with 9999 value